webapp/components/survey/SurveyCreate/store/actions/useOnCreate.js

```diff
@@ -1,21 +1,45 @@
 import axios from 'axios'
 import { useDispatch } from 'react-redux'
 
+import * as Authorizer from '@core/auth/authorizer'
+
 import { SurveyActions } from '@webapp/store/survey'
 import { JobActions } from '@webapp/store/app'
 import * as JobSerialized from '@common/job/jobSerialized'
+import { NotificationActions } from '@webapp/store/ui'
+import { useUser } from '@webapp/store/user'
+
+const sendSurveyCreateRequest = async ({ dispatch, newSurvey, user }) => {
+  const { name, label, lang, cloneFrom = false, template = false } = newSurvey
+  try {
+    const { data } = await axios.post('/api/survey', { name, label, lang, cloneFrom, template })
```

Comment:
maybe this can be. moved to the API folder

Reply:
uhm, right!

Reply:
I've moved it partially into API... using NotificationActions there creates a dependency loop